### ISSUE TITLE
Error on fresh install: Cannot find module 'concat-stream'

### ISSUE THREAD

**Issue by branneman** [anko/eslisp#58]
Steps to reproduce:
```sh
mkdir esl
cd esl
npm init -y
npm i eslisp
echo '((. console log) "Yo!")' | npx eslc
```

Output:
```
Cannot find module 'concat-stream'
```

Workaround:
```sh
npm i concat-stream
```

Potential fix: this seems like a simple missing dependency, which should just be added to the `package.json`.

**Comment by anko** [comment]
Oops.

It's in `devDependencies`, so the tests are passing.  But it should be in `dependencies`, so the package would work.

**Comment by anko** [comment]
Published in v0.8.2.

**Comment by branneman** [comment]
:tada:
